### PR TITLE
Upstream issue 1042

### PR DESCRIPTION
**Pull request by jfg956**
Related issue: https://github.com/github/orchestrator/issues/1042

This PR adds documentation for missing parameters (issues 1041).

This PR also does a little more than above issue to make things a little better:
- I replace some legacy names in config files by updated naming,
- and I make sample configuration files a little more consistent with each-others.

### CODE REVIEW

**Review by shlomi-noach — APPROVED** [review]
Thank you!

**Comment by shlomi-noach** [comment]
closes #1042

File: conf/orchestrator-simple.conf.json [hunk 1]
```diff
@@ -3,38 +3,58 @@
...
+  "SkipBinlogServerUnresolveCheck": true,
+  "ExpiryHostnameResolvesMinutes": 60,
+  "RejectHostnameResolvePattern": "",
   "ReasonableReplicationLagSeconds": 10,
   "ProblemIgnoreHostnameFilters": [],
+  "VerifyReplicationFilters": false,
+  "ReasonableMaintenanceReplicationLagSeconds": 20,
+  "VerifyReplicationFilters": false,
+  "ReasonableMaintenanceReplicationLagSeconds": 20,
+  "CandidateInstanceExpireMinutes": 60,
```
Comment (EagleEyeJohn) [inline conf/orchestrator-simple.conf.json:53]: these lines are duplicated?

Comment (shlomi-noach) [reply conf/orchestrator-simple.conf.json:53]: @EagleEyeJohn good catch. Would either you or @jfg956 be interested to submit a pull request to fix this?

**Pull request merged**